### PR TITLE
prettier

### PR DESCRIPTION
**Pull request by JakeyGilly**

### CODE REVIEW

**Comment by dandabs** [comment]
Please update PR name, commit name and ticket assignment to conform to the repository's code change standards as per https://verglas.atlassian.net/wiki/x/DgAN.

Also see requested changes on previous review.

File: packages/frontend/README.md [hunk 1]
```diff
@@ -4,47 +4,47 @@ This template provides a minimal setup to get React working in Vite with HMR and
 
 Currently, two official plugins are available:
 
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+-   [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
```
Comment (dandabs) [inline packages/frontend/README.md:7]: why is this spacing not consistent with the 4 spaces elsewhere?

File: packages/frontend/public/vite.svg
Comment (dandabs) [inline packages/frontend/public/vite.svg:1]: what changes in this file?

**Comment by dandabs** [comment]
Prettier check failing

**Comment by BobbyShmurner** [comment]
> Prettier check failing

Seems to be unrelated to the pr, the action workflow just isn't configured correctly
https://github.com/VerglasSoftware/educatr-monorepo/actions/runs/11827497580/job/32955752564?pr=2

**Comment by JakeyGilly** [comment]
i knew it

**Comment by dandabs** [comment]
> > Prettier check failing
> 
> Seems to be unrelated to the pr, the action workflow just isn't configured correctly https://github.com/VerglasSoftware/educatr-monorepo/actions/runs/11827497580/job/32955752564?pr=2

I see – should maybe be fixed in this PR as its related to prettier anyway

**Pull request closed without merging**